pkg/client/didexchange/client_test.go

```diff
@@ -7,9 +7,13 @@ SPDX-License-Identifier: Apache-2.0
 package didexchange
 
 import (
+	"encoding/json"
+	"errors"
 	"fmt"
 	"testing"
+	"time"
 
+	"github.com/btcsuite/btcutil/base58"
```

Comment:
what uses base58?

Reply:
It was part of a test case. [Removed](https://github.com/hyperledger/aries-framework-go/compare/0bd1de40ac4074ba010d839b6601c3c66fa10c93..ddd9fb8148ffdcadbedd52260b1d70fa48776bdc#diff-415ff83c536dcba8813f0209c1104a1dL16) it as it's not really required.